hoomd/ExecutionConfiguration.cc

```diff
@@ -427,6 +423,7 @@ void ExecutionConfiguration::printGPUStats()
         int mib = int(float(m_dev_prop[idev].totalGlobalMem) / float(1024*1024));
         s << ", " << setw(4) << mib << " MiB DRAM";
 
+        #if defined(__HIP_PLATFORM_NVCC__) && 0 // disabled for now
```

Comment:
Is is possible to re-enable this when the backend is NVCC?

Reply:
re-enabled